test/TestDelegateProxy.sol

```diff
@@ -0,0 +1,75 @@
+pragma solidity 0.4.18;
+
+import "truffle/Assert.sol";
+import "./helpers/ThrowProxy.sol";
+
+import "../contracts/common/DelegateProxy.sol";
+import "../contracts/evmscript/ScriptHelpers.sol";
+
+
+contract Target {
+    function returnSomething() public pure returns (bool) { return true; }
+    function dontReturn() {}
+    function fail() { revert(); }
+    function die() { selfdestruct(0); }
+}
+
+
+contract TestDelegateProxy is DelegateProxy {
+    using ScriptHelpers for *;
+
+    Target target;
+    ThrowProxy throwProxy;
+
+    function beforeAll() {
+        target = new Target();
+    }
+
+    function beforeEach() {
+        throwProxy = new ThrowProxy(address(this));
+    }
+
+    function testMinReturn0WithoutReturn() {
+        delegateFwd(target, target.dontReturn.selector.toBytes(), 0);
+    }
+
+    function testMinReturn0WithReturn() {
+        delegateFwd(target, target.returnSomething.selector.toBytes(), 0);
```

Comment:
For these that don't fail, it'd be great if we could assert that their returns are the same as what `Target` should've returned (or nothing). But doing this would probably be easier in a separate JS-based test for the `DelegateProxy`.

Reply:
yep